episodes/17-live.md

```diff
@@ -1,51 +1,67 @@
 ---
 start: no
-title: Live Coding is a Skill
+title: Live Demonstration is a Skill
 block: Building Teaching Skill
 teaching: 20
 exercises: 45
 ---
 
 One of the cornerstones of The Carpentries teaching is live
-coding: *instructors do not use slides to teach coding*, but work through the lesson material,
-typing in the code or instructions, with the workshop participants following
+demonstrations: *instructors do not use slides to teach*, but work through the lesson material,
+typing in code or modeling procedures, with the workshop participants following
 along. This section explains how it works, why we use it, and
-gives general tips for an effective participatory live coding presentation. We will
+gives general tips for an effective participatory live demo presentation. We will
 finish this section by practicing ourselves and providing feedback for each other.
 
+
 ::::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain the advantages and limitations of participatory live coding.
-- Summarize the key dos and do nots of participatory live coding.
-- Demonstrate participatory live coding.
+- Explain the advantages and limitations of participatory live demos.
+- Summarize the key dos and do nots of participatory live demos.
+- Demonstrate participatory live demos.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
 :::::::::::::::::::::::::::::::::::::::: questions
 
-- Why do we teach using participatory live coding?
+- Why do we teach using participatory live demos?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-## Why Participatory Live Coding?
+## Why Participatory Live Demos?
 
 We do not use slides in our lessons.
-Instead, instructors plug their laptop into the projector
+Instead, instructors share their computer screens
 and work through the lesson,
 typing in the code,
 reformatting data,
+demonstrating commands and procedures,
 and talking as we go.
-This is called ["live coding"](https://en.wikipedia.org/wiki/Live_coding).
+This is called "live coding" or more generally "live demonstration".
 However, the instructor is not live coding in a vacuum.
 Importantly, learners are strongly encouraged
 to "code-along" with the instructor.
 We refer to the practice of having the instructor live code and the learners code along as "participatory live coding" or, less formally, 'code-along sessions'.
 
+:::::::::::::::::::::::::::::::::::::::::  callout
+
+## Live Coding or Live Demonstration?
+
+The term "live coding" is used throughout instructor training
+to describe the Carpentries model of teaching,
+but you don't need to be a programmer or know how to code at all
+to integrate live demonstrations in your teaching.
+Many of our lessons on tools like spreadsheets and OpenRefine
+include no coding at all, and similar models are widely used
```

Comment:
This is not entirely correct, although I see the distinction of these two lessons. Both use some use of functions. Might change it to "Some lessons might not include coding..."